src/CapabilityService.WebApi/Startup.cs

```diff
@@ -139,8 +143,7 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             app.UseHttpMetrics();
 
             app.UseMvc();
-
-
+            
```

Comment:
In older version of their NuGet package there was indeed a need to call app.UserAuthentication. But it seems with the new AzureADBearer it is either enabled by default or it doesnt use middleware at all for the JWT authentication (in theory it could be handled by a filter). You can read more about how to enable the middleware  @ https://docs.microsoft.com/en-us/azure/active-directory/develop/scenario-protected-web-api-app-configuration